libraries/GCS_MAVLink/GCS_MAVLink.cpp

```diff
@@ -79,7 +79,7 @@ void GCS_MAVLINK::set_channel_private(mavlink_channel_t _chan)
 }
 
 // return a MAVLink variable type given a AP_Param type
-uint8_t mav_var_type(enum ap_var_type t)
+mavlink_message_type_t mav_var_type(enum ap_var_type t)
```

Comment:
It should actually be MAV_PARAM_TYPE (and the returns should be MAV_PARAM_TYPE_INT8, etc). It returns the same values at the moment, but might change in the future.